discord/enums.py

```diff
@@ -639,7 +639,7 @@ class GuildEventStatus(Enum):
     scheduled = 1
     active = 2
     completed = 3
-    canceled = 4
+    cancelled = 4
```

Comment:
> I think that’s what it is

Interesting, I didn't know "canceled"  was also a word (just googled it, and it is) but I think aliasing is always an option, like `discord.Color` and `discord.Colour` :grin:

Reply:
yeah probably